akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/BehaviorTestKit.scala

```diff
@@ -4,24 +4,32 @@
 
 package akka.actor.testkit.typed.scaladsl
 
-import java.util.concurrent.ThreadLocalRandom
-
-import scala.collection.immutable
-import scala.reflect.ClassTag
-
+import akka.actor.testkit.typed.internal.{ ActorSystemStub, BehaviorTestKitImpl }
 import akka.actor.testkit.typed.{ CapturedLogEvent, Effect }
-import akka.actor.testkit.typed.internal.BehaviorTestKitImpl
-import akka.actor.typed.{ ActorRef, Behavior, Signal, TypedActorContext }
 import akka.actor.typed.receptionist.Receptionist
+import akka.actor.typed.{ ActorRef, Behavior, Signal, TypedActorContext }
 import akka.annotation.{ ApiMayChange, DoNotInherit }
+import com.typesafe.config.Config
+
+import java.util.concurrent.ThreadLocalRandom
+import scala.collection.immutable
+import scala.reflect.ClassTag
 
 @ApiMayChange
 object BehaviorTestKit {
-  import akka.actor.testkit.typed.scaladsl.TestInbox.address
 
-  def apply[T](initialBehavior: Behavior[T], name: String): BehaviorTestKit[T] = {
+  trait config {
+    def appTestConfig: Config = ActorTestKit.ApplicationTestConfig
+  }
+  object config extends config
```

Comment:
Just make it a method on BehaviorTestKit without the trait and extra object. Should perhaps be `private[akka]` as well as it is meant for sharing between internals rather than for users to access?

Reply:
actually the idea was to expose a 'pre-baked' config object, perhaps some more in the future.
notice that for this specific PR we keep backward compatibility, hence creating a test-kit without explicitly providing a config object will resort to the reference config. however users requiring control over the config object must provide one, the intention behind this is to provide them a reasonable default for the common case.
so (in other words) this is definitely not akka internal as its meant to be used by user code

Reply:
`ActorTestKit.ApplicationTestConfig` already is public if users want to combine the `application-test.conf` with something (without referencing the file name themselves), so I don't think there is any value for an additional path to the same value.

Reply:
but there must be a way to point users at the 'right direction', some kind of a hint towards the best practice.
I think since at this point the default is not adequate, there should at least be an exposed pre-baked object satisfying the best practice

Reply:
I don't find that very important, given that it is open source and anyone can look at what the default apply method does. In most cases a behavior that is isolated enough to be tested with the behavior testkit will not touch more than "its own" config, for that it is likely most often sufficient with just supplying a non-loaded-config object with the specific config block in place.

Adding a note in the Scala/Javadoc either for the default apply to detail that it uses the test-application.conf would be good though.